app/views/sessions/_card.html.erb

```diff
@@ -1,35 +1,67 @@
-<div class="w-full p-5 bg-white rounded-lg">
-    <div class="flex flex-row items-center mb-6">
-      <% if speaker.profile.image.attached? %>
-        <%= image_tag url_for(profile.image), class: "text-red w-12 h-12 rounded-full mr-4" %>
-      <% else %>
-        <div class="flex flex-row items-center justify-center w-12 h-12 p-2 mr-4 rounded-full min-w-12 min-h-12 bg-slate-300">
-            <%= image_tag url_for("user_avatar.svg") %>
+<%# TODO: remove when we add support for multiple speakers %>
+<% speaker = session.speakers.first %>
+
+<div class="flex flex-col w-full p-5 mb-4 bg-white rounded-[20px]">
+  <div class="flex flex-col w-full">
+    <div class="flex justify-between w-full mb-4">
+      <div class="flex">
+        <% if speaker&.image&.attached? %>
+          <%= image_tag url_for(speaker&.image), class: "text-red w-12 h-12 rounded-full mr-4" %>
+        <% else %>
+          <div class="flex flex-row items-center justify-center w-12 h-12 p-2 mr-4 rounded-full min-w-12 min-h-12 bg-slate-300">
+            <%= inline_svg_tag "icons/user_avatar.svg" %>
+          </div>
+        <% end %>
+
+        <div class="flex flex-col justify-center h-full">
+          <a href="<%= speaker_path(speaker) %>">
+            <h2 class="italic font-black text-red"><%= speaker&.name %></h2>
```

Comment:
Seems like the page becomes unusable if at least one session doesn't have a speaker. I think we shouldn't show "incomplete" sessions, that is, those without a location or a speaker. (`@sessions = sessions.joins(:speakers, :location).includes(:attendees, :tags)`
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/4b0bb54d-2b48-4a25-b4ec-135bf16f312e">